mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -277,7 +277,7 @@ func schemaNodeConfig() *schema.Schema {
 				"tags": {
 					Type:     schema.TypeList,
 					Optional: true,
-					ForceNew: true,
+					ForceNew: false,
```

Comment:
`ForceNew` is set to `false` as default and we don't need to set it explicitly.

Reply:
Fixing now